bot/__main__.py

```diff
@@ -48,7 +48,7 @@
 
 # Local setups usually don't have the clickup key set,
 # and loading the cog would simply spam errors in the console.
-if CLICKUP_KEY is not None:
+if ClickUp.key is not None:
     bot.load_extension("bot.cogs.clickup")
 else:
     log.warning("`CLICKUP_KEY` not set in the environment, not loading the ClickUp cog.")
```

Comment:
I wouldn't consider not having ClickUp key a warning, as I don't expect many, if not any, local setups to have a valid ClickUp Key, info should be fine here

Reply:
This is from another PR. But sure